src/integrationTest/java/com/mongodb/hibernate/SessionFactoryTests.java

```diff
@@ -16,7 +16,7 @@
 
 package com.mongodb.hibernate;
 
-import static org.hibernate.cfg.JdbcSettings.JAKARTA_JDBC_URL;
+import static org.hibernate.cfg.AvailableSettings.JAKARTA_JDBC_URL;
```

Comment:
Let's always refer to the out-of-the-box configuration properties via `AvailableSettings`, as all of them are available through this interface.

Reply:
agree. The original reason to use `JdbcSettings` is to emphasize the relevant sub scope.